server/util/db.js

```diff
@@ -12,7 +13,7 @@ if (String(process.env.DEBUG).includes('sql') || process.env.SQL_DEBUG) {
     };
 }
 
-const pool = new Pool();
+const pool = new Pool(getDbConfig());
```

Comment:
This change ensures that this instantiation of `Pool` is always using the config that is produced by `getDbConfig` and does not rely on implicit default `$PG*` var access.